core/src/main/scala/anorm/Column.scala

```diff
@@ -280,6 +280,19 @@ object Column extends JodaColumn with JavaTimeColumn {
     }
   }
 
+  private[anorm] val columnToBooleanAcceptingByte: Column[Boolean] = nonNull { (value, meta) =>
```

Comment:
```suggestion
  private[anorm] val columnByteToBoolean: Column[Boolean] = nonNull { (value, meta) =>
```

Reply:
I performed the renaming as requested, even though I find it slightly misleading (the conversion works for boolean datatype as well, it is not limited to byte or short). I added a line of documentation to clarify this.